### ISSUE TITLE
era-beta: search results page

### ISSUE THREAD

**Issue by sfarnel** [ualbertalib/jupiter#507]
- the subject filter appears twice (Subject/Keyword and Subject); just Subject on wireframe
- Sort Year facet in wireframe is just Year
- Sort Year seems to be ordered by number of items rather than reverse chronological
- Author/Creator/Contributor on wireframe is simply Author
- is there a way to have Article (Draft / Submitted)
- Visibility showing as filter although not in wireframe?

**Comment by cwant** [comment]
Thanks Sharon! I've broken off the subject facet issue as #510, as it is clearly a bug. (The other ones will need a closer look.)

**Comment by sfarnel** [comment]
Thanks Chris!

**Comment by murny** [comment]
There is still work to do on this one. 

- [ ] Sort Year facet in wireframe is just Year
- [ ] Author/Creator/Contributor on wireframe is simply Author
- [x] Visibility showing as filter although not in wireframe? (we limit this to admin only, so shouldn't be an issue now)

Assuming this is being worked on, leaving this open

**Comment by weiweishi** [comment]
Yes, the only remaining task on this issue is:
- [ ] Sort Year facet in wireframe is just Year

**Comment by murny** [comment]
Bullet point 2 is still validate? We still set author as "Author / Creator / Contributor"? or was there a decision to leave this as is?

![image](https://user-images.githubusercontent.com/1930474/39325367-30d13cb2-494f-11e8-92e4-a6053b854595.png)

**Comment by weiweishi** [comment]
Yes, a discussion with Sonya and Leah this morning confirmed that they are
okay with Author / Creator / Contributor as-is. Will update wireframe to
reflect this.


*Weiwei Shi*Digital Initiative Applications Librarian

2-10L Cameron Library, University of Alberta
780-492-7802  |  weiwei.shi@ualberta.ca
*"The University of Alberta respectfully acknowledges that we are situated
on Treaty 6 territory, traditional lands of First Nations and Métis
people."*
